### ISSUE TITLE
`block-closing-brace-newline-before` and `block-closing-brace-empty-line-before` mark a column past the closing brace where a stray semicolon stands behind it

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#735]
### What minimal example or steps are needed to reproduce the bug?

PostCSS keeps a stray semicolon behind a rule's closing brace, with the whitespace in front of it, in the rule's `raws.ownSemicolon` and prints that raw behind the brace. Both rules count their report index from the end of the printed node, `nodeString(statement, result).length - 2` and `- 1`, so the index lands inside the raw rather than on the brace:

```css
a {
	b {
		c: d }
	;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/block-closing-brace-newline-before": "always",
		"@stylistic/block-closing-brace-empty-line-before": "always-multi-line"
	}
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json
```

### What did you expect to happen?

Each warning to mark the closing brace, or the character in front of it as the rules do elsewhere, whatever the file writes behind the brace.

### What actually happened?

| rule | input | reported | without the semicolon |
| --- | --- | --- | --- |
| `block-closing-brace-newline-before`, `always` | `a {⏎\tb {⏎\t\tc: d }⏎\t;⏎}⏎` | `4:1` | `3:7` |
| `block-closing-brace-empty-line-before`, `always-multi-line` | `a {⏎\tb {⏎\t\tc: d⏎\t}⏎\t;⏎}⏎` | `5:2` (and `6:1` for the outer block) | `4:2` |

The message and the fix are right; only the position is off, by the length of the raw. Measured on `b3409da` and on the branch of #562 alike, under `postcss`, `postcss-scss` and `postcss-less`.

### What is the cause?

The printed copy of the rule ends on `raws.ownSemicolon`. `statementString` (`lib/utils/statementString/index.ts`, added for #562) prints a statement through its closing brace, and `block-closing-brace-space-before` counts its index from it already.

### Do you have a proposal to fix the bug?

Count both indices from `statementString` instead of `nodeString`, as #562 did for `block-closing-brace-space-before`.

### Notes

Found while fixing #562. #568 is the same shape met by `indentation`, and the same answer.
